anime_downloader/cli.py

```diff
@@ -4,6 +4,7 @@
 
 import logging
 
+from anime_downloader.sites.masterani import Masterani
```

Comment:
Remove this import as you won't need it anymore after the change mentioned below.